File: code-coach/free/easy/halloween_candy.java
import java.util.Scanner;

public class Program {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int houses = input.nextInt();

        // your code goes here
        double value = 2 * 100.0 / houses;
        System.out.println((int)Math.ceil(value));
    }
}

File: code-coach/free/easy/halloween_candy.cpp
#include <iostream>
#include <cmath>

using namespace std;
int main() {
    int houses;
    double value;

    cin >> houses;

    // your code goes here
    value = 2 * 100.0 / houses;
    cout << static_cast<int>(ceil(value)) << endl;

    return 0;
}